Geometry/HGCalGeometry/src/HGCalGeometry.cc

```diff
@@ -315,38 +317,93 @@ HGCalGeometry::CornersVec HGCalGeometry::getCorners(const DetId& id) const {
   return co;
 }
 
-HGCalGeometry::CornersVec HGCalGeometry::get8Corners(const DetId& id) const {
+HGCalGeometry::CornersVec HGCalGeometry::get8Corners(const DetId& detid) const {
   
   unsigned int ncorner = FlatTrd::ncorner_;
   HGCalGeometry::CornersVec co (ncorner, GlobalPoint(0,0,0));
-  unsigned int cellIndex =  indexFor(id);
+  unsigned int cellIndex =  indexFor(detid);
   if ((cellIndex <  m_cellVec.size() && m_det != DetId::HGCalHSc) ||
       (cellIndex <  m_cellVec2.size() && m_det == DetId::HGCalHSc)) {
-    HGCalTopology::DecodedDetId id_ = m_topology.decode(id);
+    HGCalTopology::DecodedDetId id = m_topology.decode(detid);
     std::pair<float,float> xy;
```

Comment:
@Mourtz - this is exact code duplication from line 281. Please, implement it as a separate private member function.

Reply:
@Mourtz - I do not see where it was resolved.